apps/src/templates/progress/ProgressBubble.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { PropTypes } from 'react';
+import Radium from 'radium';
+import color from "@cdo/apps/util/color";
+
+export const DOT_SIZE = 30;
+
+// TODO - not sure I like these being keyed by these strings.
```

Comment:
Now seems like as good a time as any to switch to an enum and computed property names, e.g.
```
const bubbleColors = {
  [Progress.SUBMITTED]: { ... },
  [Progress.PERFECT]: { ... },
```